src/deadline/client/cli/_deadline_cli.py

```diff
@@ -76,3 +77,4 @@ def main(ctx: click.Context, log_level: str):
 main.add_command(cli_job)
 main.add_command(cli_queue)
 main.add_command(cli_worker)
+main.add_command(cli_manifest)
```

Comment:
Is this CLI ready to be used by public? Do we consider a feature flag to stage and mature this, especially some commands are not implemented yet.

Reply:
Agreed! Incorporated similar to your PR.